cnxarchive/scripts/hits_counter.py

```diff
@@ -93,14 +93,18 @@ def main(argv=None):
     with psycopg2.connect(connection_string) as db_connection:
         with db_connection.cursor() as cursor:
             for ident_hash, hit_count in hits.items():
-                cursor.execute(SQL_GET_MODULE_IDENT_BY_UUID_N_VERSION,
-                               split_ident_hash(ident_hash))
-                module_ident = cursor.fetchone()
-                payload = (module_ident, start_timestamp, end_timestamp,
-                           hit_count,)
-                cursor.execute("INSERT INTO document_hits "
-                               "  VALUES (%s, %s, %s, %s);",
-                               payload)
+                id, version, id_type = split_ident_hash(ident_hash)
```

Comment:
If we make the `URL_PATTERN_TMPLT` stricter to match against uuids, we won't need to check the type.  (and I think we should only match the urls that use uuids.)